search-app/test/cmr/search/test/unit/services/query_execution.clj

```diff
@@ -3,116 +3,143 @@
   queries are given the correct execution strategy."
   (:require
    [clojure.test :refer :all]
+   [cmr.common.hash-cache :as hc]
    [cmr.elastic-utils.search.es-params-converter :as pc]
    [cmr.elastic-utils.search.query-execution :as qe]
+   [cmr.metadata-db.data.concepts :as c]
    [cmr.search.services.query-execution :as search-qe]))
 
+(def test-context
+  {:system
```

Comment:
Why did you create this?  Were you getting a hashcache protocal for nil class error?  Or is there some other reason?  Or is this so you don't need to spin up an embedded redis server fixture like we do in a lot of other tests?

Reply:
This is needed to make the provider hash work with the test. Since this is unit test, we don't have any real system component set up in the test.